.eslintrc.js

```diff
@@ -2,11 +2,12 @@ module.exports = {
     env: {
         browser: true,
         es2021: true,
-        "jest/globals": true,
+        'jest/globals': true,
     },
     extends: [
         'airbnb-base',
         'plugin:import/typescript',
+        'plugin:jest/recommended',
```

Comment:
Ah, this explains why I had an error before (which I promptly forgot about :smile: )